src/Entity/MetadataDisplayEntity.php

```diff
@@ -429,29 +432,35 @@ public function getTwigVariablesUsed() {
    * @param \Twig\Node\ModuleNode $nodes
    *   A Twig Module Nodes object.
    *
+   * @param \Twig\Node\BodyNode $nodes
+   *   A Twig Module Nodes object.
+   *
+   * @param \Twig\Node\Node $nodes
+   *   A Twig Module Nodes object.
+   *
    * @return array
    *   A list of used $variables by this template.
    */
-  private function getTwigVariableNames(ModuleNode $nodes): array {
+  private function getTwigVariableNames(ModuleNode|Node|BodyNode $nodes): array {
```

Comment:
Ohhh.. this is new to me! So cool. So 3 params in the docs but really a single multi type one in the function argument?